Segments/Annwn.mapproj

```diff
@@ -62958,11 +63522,642 @@
+  
+    return wizard;
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+      <script name="OnDeath" enabled="false">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+      <script name="OnIncomingPlayer" enabled="false">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+    </entity>
+    <entity name="level15Minotaur">
+      <script name="OnSpawn" enabled="true">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+	var minotaur = new Minotaur()
+    {
+        MaxHealth = 300, Health = 300,
+        BaseDodge = 27,
+        HideDetection = 32, 
+        Experience = 11000,
+        MagicProtection = 17,
+        CanCharge = true,
+        BasePenetration = ShieldPenetration.Medium,
+        Immunity = CreatureImmunity.Web
+    };
+    
+    minotaur.AddStatus(new NightVisionStatus(minotaur));
+    
+    minotaur.Attacks = new CreatureAttackCollection()
+    {
+        { 
+            new CreatureAttack(16,     30, 40,      "The minotaur hits with a wicked axe"),     60 
+        }, 
+        {
+            new CreatureAttack(16,     35, 45,     "You've been gored by the minotaur",
+                                                    new AttackStunComponent(20.0)),             30
+        }, 
+        {
+            new CreatureAttack(16,     55, 60,     "You've been trampled by the minotaur"),     10
+        }, 
+    };
+    
+    minotaur.Blocks = new CreatureBlockCollection()
+    {
+        new CreatureBlock(11, "the armor"),
+        new CreatureBlock(12, "its axe")
+    };
+    
+    minotaur.Wield(new BattleAxe());    
+    
+    minotaur.AddGold(160);
+  
+    return minotaur;
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+      <script name="OnDeath" enabled="false">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+      <script name="OnIncomingPlayer" enabled="false">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+    </entity>
+    <entity name="level15Archer">
+      <script name="OnSpawn" enabled="true">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+	var archer = new Humanoid()
+    {
+        Name = _names.Random(),
+        Body = 32,
+        HideDetection = 32,    
+        MaxHealth = 280, Health = 280,
+        BaseDodge = 27,
+        BasePenetration = ShieldPenetration.Medium,
+        Experience = 10500,
+        
+        CanFlee = true,
+        CanSwim = true,
+    };
+    
+    archer.Attacks = new CreatureAttackCollection
+    {
+        new CreatureBasicAttack(16)
+    };
+
+    archer.Wield(new Crossbow());
+    archer.Equip(new PlatemailArmor());
+
+    archer.AddGold(146);
+  
+    return glamuzu;
```

Comment:
return archer?

Reply:
lol typo on the typo correction return archer;

Reply:
Fixed